app/models/manageiq/providers/ibm_cloud/power_virtual_servers/manager_mixin.rb

```diff
@@ -37,17 +37,9 @@ def connect(options = {})
   end
 
   def api_endpoint_url(location)
-    api_endpoint_overrides = ::Settings.ems.ems_ibm_cloud_power_virtual_servers.api_endpoint_overrides
-
-    if api_endpoint_overrides.key?(location.to_sym)
-      url = api_endpoint_overrides[location.to_sym]
-    else
-      region = location.sub(/-\d$/, '')
-      region = region.sub(/\d\d$/, '')
-      url = "#{region}.power-iaas.cloud.ibm.com"
-    end
-
-    url
+    region = location.sub(/-\d$/, '')
+    region = region.sub(/\d\d$/, '')
```

Comment:
An example in a comment of the input `location` and how `region` looks after these substitutions would be helpful

Reply:
Here's the type of comment I was thinking:
```ruby
    region = location.sub(/-\d$/, '')  # e.g. 'eu-de-2' -> 'eu-de'
    region = region.sub(/\d\d$/, '')   # e.g. 'syd04' -> 'syd'
```